docs/abi-spec.rst

```diff
@@ -78,13 +78,13 @@ Types can be combined to a tuple by enclosing them inside parentheses, separated
 
 It is possible to form tuples of tuples, arrays of tuples and so on.  It is also possible to form zero-tuples (where ``n == 0``).
 
-.. note::
-    Solidity supports all the types presented above with the same names with the exception of tuples. The ABI tuple type is utilised for encoding Solidity ``structs``.
-
 Mapping Solidity to ABI types
 -----------------------------
-The following table shows on the left column Solidity types that are not part of the ABI,
-and on the right column the ABI type that they map to.
+
+Solidity supports all the types presented above with the same names with the
+exception of tuples. On the other hand, some Solidity types are not supported
+by the ABI.  The following table shows on the left column Solidity types that
+are not part of the ABI, and on the right column the ABI type that they map to.
```

Comment:
Perhaps "the ABI types that are used in their place"? It might not be 100% clean what "map to" means and when this mapping is applied.

Reply:
Solidity type X is represented by ABI type Y. Or encoded by ABI type Y?

Reply:
updated